src/tools/_version.py

```diff
@@ -1,2 +1,2 @@
 """This file defines the version of this module."""
-__version__ = "0.0.6"
+__version__ = "0.0.6-rc.1"
```

Comment:
Does this need to be updated or is the rc.1 tag valid here?

Reply:
It should be finalized.

Reply:
👍 referencing the commit:

resolved in 7ed1f98ec5c4396ffcdf131635af500f38627962